src/test/resources/software/amazon/smithy/lsp/ext/models/main.smithy

```diff
@@ -0,0 +1,44 @@
+$version: "1.0"
```

Comment:
Might be worth throwing some oddly formatted definitions in here just to make sure they work as well - indented, same line closing braces, etc.

Reply:
Added in 7aa59d0